kuma/javascript/src/payments/pages/terms.jsx

```diff
@@ -0,0 +1,116 @@
+// @flow
+import * as React from 'react';
+import SubHeader from '../components/subheaders/index.jsx';
+
+type Props = {
+    data: {
+        contributionSupportEmail: string,
+    },
+};
+
+export const title = 'Payment Terms';
+const TermsPage = ({ data: { contributionSupportEmail } }: Props) => (
+    <>
+        <SubHeader title={title} />
+        <main
+            className="contributions-page"
+            role="main"
+            data-testid="terms-page"
+        >
+            <section>
+                <p>Mozilla (that’s us) operates MDN Web Docs.</p>
+                <p>
+                    These Payment Terms cover your payment to MDN.{' '}
+                    <a
+                        href="https://www.mozilla.org/en-US/about/legal/terms/mozilla/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        The Websites & Communications Terms of Use
+                    </a>{' '}
+                    cover the MDN service itself. If there is any inconsistency
+                    between these terms and the Websites & Communications Terms
+                    of Use, these Payment Terms apply.
+                </p>
+
+                <h2>Payment Authorization</h2>
+                <p>
+                    You can support MDN with a one-time payment, or you can
+                    choose to schedule monthly payments. Your payments are not
+                    tax deductible.
+                </p>
+                <p>
+                    In order to sign up for monthly payments, you must sign in
+                    to MDN using a GitHub account. GitHub has its own{' '}
+                    <a
+                        href="https://help.github.com/articles/github-terms-of-service/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Terms
+                    </a>{' '}
+                    and{' '}
+                    <a
+                        href="https://help.github.com/articles/github-privacy-statement/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        Privacy Policy
+                    </a>
+                    .
+                </p>
+                <p>
+                    When you choose to support MDN, you authorize us to charge
+                    the payment card you provide for the amount you choose,
+                    either as a one-time payment, or as a recurring monthly
+                    payment.
+                </p>
+                <h2>Cancellation</h2>
+                <p>
+                    If you sign up for monthly payments, you may cancel at any
```

Comment:
There is no other option :) I don't think changing the copy in this PR is necessary but, it is good that we call these instances out so it is on our radar.

Reply:
I am glad you read this! I just copied and pasted. Made an issue for these https://github.com/mdn/kuma/issues/6889